src/aap_eda/services/activation/manager.py

```diff
@@ -1053,6 +1053,7 @@ def _create_activation_instance(self):
             "name": self.db_instance.name,
             "status": ActivationStatus.STARTING,
             "git_hash": git_hash,
+            "organization": self.db_instance.organization,
```

Comment:
Calling it out in a comment, this is a case where the child inherits the organization from its parent